sys-tuner/src/main.rs

```diff
@@ -93,6 +93,10 @@ fn tune_kernel_udp_buffers_and_vmmap() {
 
     // increase mmap counts for many append_vecs
     sysctl_write("vm.max_map_count", "1000000");
+
+    // Reference: https://community.mellanox.com/s/article/linux-sysctl-tuning
```

Comment:
This document is referencing TCP tuning, so I poked around for the same vars WRT UDP and found https://indico.cern.ch/event/212228/contributions/1507212/attachments/333941/466017/10GE_network_tests_with_UDP.pdf.  Admittedly it's tuning 10Gbit and using larger MTU, but at least the right protocol.  The suggestions are quite a bit different and also tough a couple other knobs that we haven't investigated.

Additionally, it got me wondering whether we might be pinning PoH to the NIC's IRQ affinity core, which would probably be bad.

WDYT about rolling new values out to our nodes for a week and monitoring before we change the code?

Reply:
Yeah, let's try on our nodes first.